src/www/ui/core-auth.php

```diff
@@ -266,15 +305,15 @@ function OutputOpen()
    *
    * @return boolean
    */
-  function checkUsernameAndPassword($userName, $password)
+  function checkUsernameAndPassword($userName, $password, $oAuthClient=false)
```

Comment:
A function called `checkUsernameAndPassword` should not modify the global state e.g. `$_SESSION`